dadguide/database_manager.py

```diff
@@ -275,22 +267,9 @@ class DgAwakening(DadguideItem):
     def __init__(self, item, database, **kwargs):
```

Comment:
are we still not able to delete self._database?

Reply:
not done yet omg i wanted u to comment about where the functions are, in graph/db_context